docs/source/markdown/options/cpus.md

```diff
@@ -0,0 +1,12 @@
+#### **--cpus**=*number*
+
+Number of CPUs. The default is *0.0* which means no limit. This is shorthand
+for **--cpu-period** and **--cpu-quota**, so you may only set either
+**--cpus** or **--cpu-period** and **--cpu-quota**.
+
+On some systems, changing the CPU limits may not be allowed for non-root
+users. For more details, see
+https://github.com/containers/podman/blob/main/troubleshooting.md#26-running-containers-with-cpu-limits-fails-with-a-permissions-error
```

Comment:
Please pick up the fix (`s/-cpu-/-resource-/`) from #15420. (And, in general, please remember that conflict resolution is very very very hard; it is a time to pay extremely close attention to what you are doing).

Reply:
thanks